### PR TITLE
Add aria hidden to alert box to prevent screen reader tab stop

### PR DESCRIPTION
**Pull request by CBonade**
## Description
Closes https://github.com/department-of-veterans-affairs/vets-design-system-documentation/issues/242

## Acceptance criteria
- [ ]

## Definition of done
- [ ] Documentation has been updated, if applicable
- [ ] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)

### CODE REVIEW

**Review by noahgelman — APPROVED** [review]
LGTM

File: src/components/va-alert/va-alert.tsx [hunk 1]
```diff
@@ -144,7 +144,7 @@ export class VaAlert {
             aria-label={this.closeBtnAriaLabel}
             onClick={this.closeHandler.bind(this)}
           >
-            <i class="fa-times-circle" aria-label="Close icon" />
+            <i aria-hidden="true" class="fas fa-times-circle" role="presentation" />
```
Comment (bkjohnson) [inline src/components/va-alert/va-alert.tsx:147]: I don't think the `fas` class is doing anything, so it should be removed. What appear to be fontawesome classes are coming from here:

https://github.com/department-of-veterans-affairs/component-library/blob/cfffd5588044c7c2f37a5bae0be625aa50f8c1af/src/components/va-alert/va-alert.css#L141-L143

**Review by bkjohnson — CHANGES_REQUESTED** [review]

**Review by 1Copenut — CHANGES_REQUESTED** [review]
👍 LGTM!

File: src/components/va-alert/va-alert.tsx (hunk 1 above)
Comment (CBonade) [reply src/components/va-alert/va-alert.tsx]: Should be all good here! Thanks for the feedback.

**Review by bkjohnson — APPROVED** [review]

**Pull request merged**